nussknacker-dist/src/universal/conf/application.conf

```diff
@@ -6,14 +6,14 @@ categoriesConfig: {
   "Default": "streaming"
 }
 
-modelClassPath: "model/genericModel.jar"
+modelClassPath: ["model/genericModel.jar"]
```

Comment:
why `model/openapi.jar` is not set in default classpath?

Reply:
It separates model in base configuration from components, also demo shows how to extend with your own components